pom.xml

```diff
@@ -6465,92 +6398,46 @@
         <pluginRepository>
-          <id>internal-releases</id>
-          <url>
-            http://mavenin.nuxeo.com/nexus/content/groups/internal-releases
-          </url>
-          <name>Nuxeo virtual release repository</name>
+          <id>maven-internal</id>
+          <url>https://packages.nuxeo.com/repository/maven-internal/</url>
           <releases>
             <enabled>true</enabled>
           </releases>
-          <snapshots>
-            <enabled>false</enabled>
-          </snapshots>
-        </pluginRepository>
-        <pluginRepository>
-          <id>internal-snapshots</id>
-          <url>
-            http://mavenin.nuxeo.com/nexus/content/groups/internal-snapshots
-          </url>
-          <name>Nuxeo virtual snapshot repository</name>
-          <releases>
-            <enabled>false</enabled>
-          </releases>
           <snapshots>
             <updatePolicy>always</updatePolicy>
             <enabled>true</enabled>
           </snapshots>
         </pluginRepository>
       </pluginRepositories>
       <distributionManagement>
-        <snapshotRepository>
-          <id>daily-snapshots</id>
-          <url>
-            http://mavenin.nuxeo.com/nexus/content/repositories/daily-snapshots
-          </url>
+        <repository>
+          <id>maven-internal-staging</id>
+          <url>https://packages.nuxeo.com/repository/maven-internal-staging/</url>
           <uniqueVersion>true</uniqueVersion>
-        </snapshotRepository>
+        </repository>
       </distributionManagement>
     </profile>
 
     <profile>
       <id>qapriv</id>
       <repositories>
         <repository>
-          <id>internal-releases</id>
-          <url>
-            http://mavenin.nuxeo.com/nexus/content/groups/internal-releases
-          </url>
+          <id>maven-internal</id>
+          <url>https://packages.nuxeo.com/repository/maven-internal/</url>
           <releases>
             <enabled>true</enabled>
           </releases>
-          <snapshots>
-            <enabled>false</enabled>
-          </snapshots>
-        </repository>
-        <repository>
-          <id>internal-snapshots</id>
-          <url>
-            http://mavenin.nuxeo.com/nexus/content/groups/internal-snapshots
-          </url>
-          <releases>
-            <enabled>false</enabled>
-          </releases>
           <snapshots>
             <enabled>true</enabled>
             <updatePolicy>always</updatePolicy>
           </snapshots>
         </repository>
         <repository>
-          <id>releases</id>
-          <url>
-            https://mavenpriv.nuxeo.com/nexus/content/repositories/releases
-          </url>
+          <id>maven-private</id>
+          <url>https://packages.nuxeo.com/repository/maven-private/</url>
           <releases>
             <enabled>true</enabled>
           </releases>
-          <snapshots>
-            <enabled>false</enabled>
-          </snapshots>
-        </repository>
```

Comment:
The `maven-private` group is harmless but not required here, as already included in `maven-internal` group.